app/controllers/concerns/auditing.rb

```diff
@@ -7,14 +7,15 @@ module Auditing
   extend ActiveSupport::Concern
 
   included do
-    _model = to_s.deconstantize.singularize
+    # called it modell cos I can't call it model
+    modell = to_s.deconstantize.singularize
```

Comment:
Perhaps call it `model_string` so that it's a) easier to follow and b) you don't need the comment.